src/brass_runner.rs

```diff
@@ -196,6 +205,20 @@ fn merge(args: &[String], context: &mut RunnerContext, task: ComputedTask) -> Fa
     Ok(())
 }
 
+fn has_user_confirmed(wasm_path: &PathBuf) -> bool {
+    println!(
+        "\nYou are about to create a Brass Golem task with the above parameters. \
+         \nThe WASM binary to be used for this task is: {:?} \
+         \nWould you like to proceed? (y/N)",
+        wasm_path
+    );
+
+    let mut input = String::new();
+    stdin().read_line(&mut input).unwrap();
```

Comment:
There is a nice crate for this purpose: [promptly](https://docs.rs/promptly/)
Check it out